DRODLib/DbXML.cpp

```diff
@@ -39,9 +39,10 @@
 #include <BackEndLib/StretchyBuffer.h>
 #include <BackEndLib/Ports.h>
 
-#include <zlib.h>
 #include <cstdio>
 
+const char gzID[] = "\x1f\x8b";
```

Comment:
Can you add a comment explaining what it is?

Reply:
Sure. Done!